.github/workflows/comment.yml

```diff
@@ -0,0 +1,62 @@
+name: Comment on clippy
+
+on:
+  workflow_run:
+    workflows: ["Rust"]
+    types:
+      - completed
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # We lock our failing tests on a known good version and
+        # informationally check the latest as well.  We should
+        # regularly update the known good once we know that tests
+        # pass on it
+        # This needs to stay in sync with rust.yml
+        rust-toolchain: [ 1.66, stable, nightly ]
```

Comment:
Can't 1.66 not generate the flag, so we don't need it here?

Reply:
By which I mean, I guess 1.66 will upload an artifact.  but we don't comment there, so why not just not run this against 1.66 and not use the artifact?

Reply:
The comment is a little weird now, since 1.67 will move in rust.yml, but is explicitly mentioned in the comment here.  Maybe s/1.67/"the known good clippy version"/?